html-forms/index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form>
+      <div>
+        <label for="email-textbox">
+          <span>Email</span>
+        </label>
+        <br />
+        <input type="email" name="email" size="20" maxlength="50" id="email-textbox" />
+      </div>
+      <div>
+        <span>Why is coding cool?</span>
```

Comment:
This should be a `label` element

Reply:
https://user-images.githubusercontent.com/72685552/136292369-479f17b0-f3b4-45fe-a9da-d673838cd5d3.mov